content/projects/gsoc/2019/gitlab-support-for-multibranch-pipeline.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: gsocproject
+title: "Multi-branch Pipeline support for Gitlab SCM"
+goal: "Add Multi-branch Pipeline support for Gitlab SCM private and SaaS instances so that Jenkins can automatically build branches and pull requests with Jenkinsfile"
+category: Plugin
+year: 2019
+status: Active
+showGoogleDoc: true
+student:
+  name: "Parichay Barpanda"
+  github: "baymac"
+mentors:
+- name: "Zhao Xiaojie (Rick)"
+  id: "linuxsuren"
+  github: "linuxsuren"
+- name: "Jeff Pearce"
+  id: "jeffpearce"
```

Comment:
The mentor list appears to be incorrect when I compare to https://summerofcode.withgoogle.com/dashboard/project/6201982094671872/overview/